src/viewer/index.ts

```diff
@@ -206,9 +210,9 @@ export class Viewer {
    *
    * @param categoryIds The identifiers of the POI categories to select.
    */
-  async selectPoiCategory(categoryIds: number[]) {
+  async selectPoiCategory(categoryId: number) {
```

Comment:
you are changing the public interface, does this change was checked against both public and internal integrators?